src/wallet/walletdb.cpp

```diff
@@ -1135,15 +1135,15 @@ std::unique_ptr<WalletDatabase> MakeDatabase(const fs::path& path, const Databas
 #ifdef USE_SQLITE
         return MakeSQLiteDatabase(path, options, status, error);
 #endif
-        error = Untranslated(strprintf("Failed to open database path '%s'. Build does not support SQLite database format.", path.string()));
+        error = Untranslated(strprintf("Failed to open database path '%s'. Build does not support SQLite database format.", fs::PathToString(path)));
```

Comment:
should not it be `fs::quoted(fs::PathToString(path))` here?

Reply:
Even in one of the more recent backports ([bitcoin#20744](https://github.com/bitcoin/bitcoin/pull/20744)), it appears that using `fs::PathToString(path)` is the norm ([source](https://github.com/bitcoin/bitcoin/blob/07269321f38e46e9e02f16d7cd135ea90692638d/src/wallet/walletdb.cpp#L1166-L1180)). Looking at even the last iterations of Bitcoin Core v23 (the version these changes first appeared), it's much the same ([source](https://github.com/bitcoin/bitcoin/blob/v23-final/src/wallet/walletdb.cpp#L1166-L1180)).